controllers/lvmcluster_controller.go

```diff
@@ -208,6 +215,46 @@ func (r *LVMClusterReconciler) reconcile(ctx context.Context, instance *lvmv1alp
 	return ctrl.Result{}, nil
 }
 
+func (r *LVMClusterReconciler) verifyLvmClusterSpec(ctx context.Context, instance *lvmv1alpha1.LVMCluster) error {
```

Comment:
method name can be more specific about what we are trying to verify instead of `verifyLvmClusterSpec`

Reply:
This comments still holds. The function only validates the device classes in the Spec. The function name should reflect that.

Reply:
added a new func for this logic and that will be called from here